programs/fall-workshops.md

```diff
@@ -11,6 +11,8 @@ Our workshop focuses on a hands-on exploration of gears and how we utilize them
 
 Each workshop has a total of 30 seats available. All students are welcome to attend any workshop, but we recommend the workshop at a high school closest to you. Students must register to attend. Should registration fill, we will place students on a waitlist in case a seat opens.
 
+<img src="/assets/events/gearworkshopflyer_final.jpg" width="500">
```

Comment:
This image overflows on small screens. If you right click on the page and press inspect, and then click "Toggle device toolbar" next to Elements, you'd see what it appears as on mobile devices. To fix, I recommend looking at [Bootstrap's responsive utilies](https://getbootstrap.com/docs/3.3/css/#responsive-utilities) for the visible-xs and hidden-xs class, or at _includes/annual-data.html for an example.
![screen shot 2017-09-25 at 3 16 43 pm](https://user-images.githubusercontent.com/20323625/30833860-a115fab8-a205-11e7-9aec-5d3c11e03a8d.png)